src/views/self/view_weapons.cpp

```diff
@@ -10,149 +10,152 @@
 			{
-				WEAPON::GIVE_DELAYED_WEAPON_TO_PED(self::ped, weapon.m_hash, 9999, false);
-			}
+				for (const auto& [_, weapon] : g_gta_data_service->weapons())
+				{
+					WEAPON::GIVE_DELAYED_WEAPON_TO_PED(self::ped, weapon.m_hash, 9999, false);
+				}
 
-			constexpr auto parachute_hash = RAGE_JOAAT("GADGET_PARACHUTE");
-			WEAPON::GIVE_DELAYED_WEAPON_TO_PED(self::ped, parachute_hash, 0, true);
-		});
-		ImGui::SameLine();
-		components::button("Remove Current Weapon", []
-		{
-			Hash weaponHash;
-			WEAPON::GET_CURRENT_PED_WEAPON(self::ped, &weaponHash, 1);
-			if (weaponHash != RAGE_JOAAT("WEAPON_UNARMED"))
+				constexpr auto parachute_hash = RAGE_JOAAT("GADGET_PARACHUTE");
+
+				WEAPON::GIVE_DELAYED_WEAPON_TO_PED(self::ped, parachute_hash, 0, true);
+			});
+			ImGui::SameLine();
+			components::button("Remove Current Weapon", []
 			{
-				WEAPON::REMOVE_WEAPON_FROM_PED(self::ped, weaponHash);
-			}
-		});
+				Hash weaponHash;
+				WEAPON::GET_CURRENT_PED_WEAPON(self::ped, &weaponHash, 1); //Should have a global var for this
 
-		ImGui::SliderFloat("Damage Multiplier", &g.weapons.increased_damage, 1.f, 10.f, "%.1f");
+				if (weaponHash != RAGE_JOAAT("WEAPON_UNARMED"))
+					WEAPON::REMOVE_WEAPON_FROM_PED(self::ped, weaponHash);
+			});
 
-		ImGui::Separator();
+			ImGui::SliderFloat("Damage Multiplier", &g.weapons.increased_damage, 1.f, 10.f, "%.1f");
 
+			ImGui::EndGroup();
+		}
+
+
+		ImGui::Separator();
 		components::sub_title("Custom Weapons");
 
+		//If I put the code below in a group to make it consitient like the rest of them I get some dumb
+		//compiler error that I don't care to fix so it's just not going in a group
 		CustomWeapon selected = g.weapons.custom_weapon;
 
 		if (ImGui::BeginCombo("Weapon", custom_weapons[(int)selected].name))
 		{
 			for (const custom_weapon& weapon : custom_weapons)
 			{
 				if (ImGui::Selectable(weapon.name, weapon.id == selected))
-				{
 					g.weapons.custom_weapon = weapon.id;
-				}
 
 				if (weapon.id == selected)
-				{
 					ImGui::SetItemDefaultFocus();
-				}
 			}
 
 			ImGui::EndCombo();
 		}
 
 		switch (selected)
 		{
+
 		case CustomWeapon::GRAVITY_GUN:
 				ImGui::Checkbox("Launch on release", &g.weapons.gravity_gun.launch_on_release);
 				break;
+
 		case CustomWeapon::VEHICLE_GUN:
 			// this some ugly ass looking code
+			// idk it just looks like the rest of the solution to me
 			static char vehicle_gun[12];
-			std::memcpy(vehicle_gun, g.weapons.vehicle_gun_model.c_str(), 12);
+			std::memcpy(vehicle_gun, g.weapons.vehicle_gun_model.c_str(), 12); //Hardcoded int?
+
 			if (ImGui::InputTextWithHint("Shooting Model", "Name of the vehicle model", vehicle_gun, sizeof(vehicle_gun)))
-			{
 				g.weapons.vehicle_gun_model = vehicle_gun;
-			}
+
 			if (ImGui::IsItemActive())
-			{
-				g_fiber_pool->queue_job([]
-				{
-					PAD::DISABLE_ALL_CONTROL_ACTIONS(0);
-				});
-			}
+				g_fiber_pool->queue_job([] {PAD::DISABLE_ALL_CONTROL_ACTIONS(0); });
```

Comment:
We don't loose if statements anymore, preferably keep the curly brackets.
(and yes I know we don't do it with curly brackets everyone in the source yet)